cmd/cosign/cli/options/verify.go

```diff
@@ -56,6 +58,13 @@ func (o *CommonVerifyOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().IntVar(&o.MaxWorkers, "max-workers", cosign.DefaultMaxWorkers,
 		"the amount of maximum workers for parallel executions")
+
+	cmd.Flags().StringVar(&o.TrustedRootPath, "trusted-root", "",
+		"Path to a Sigstore TrustedRoot JSON file.")
```

Comment:
should it also say: ` Requires --new-bundle-format to be set.`?

Reply:
Yes, good point! I pushed that change.